controllers/helper/userHelper.js

```diff
@@ -0,0 +1,328 @@
+import User from "../../models/userSchema.js";
+import {
+  getUserValidationErrors,
+  emailRegex,
+  passwordRegex,
+} from "../../utils.js";
+
+export const create = async (userData) => {
+  try {
```

Comment:
too many return statements one function should have one return statement in try and one in catch